dapp/src/pages/proposals/index.tsx

```diff
@@ -0,0 +1,250 @@
+        description
+        approvals
+        teamApproval
+        voteCount
+        createdAt
+        modifiedAt
+      }
+
+      approved: proposals(
+        where: { status: APPROVED }
+        orderBy: createdAt
+        orderDirection: desc
+      ) {
+        number
+        voteCount
+        description
+        createdAt
+        modifiedAt
+      }
+
+      rejected: proposals(
+        where: { status: REJECTED }
+        orderBy: createdAt
+        orderDirection: desc
+      ) {
+        number
+        voteCount
+        status
+        description
+        teamApproval
+        createdAt
+        modifiedAt
+      }
+
+      completed: proposals(
+        where: { status: COMPLETED }
+        orderBy: createdAt
+        orderDirection: desc
+      ) {
+        number
+        voteCount
+        status
+        description
+        createdAt
+        modifiedAt
+      }
+    }
+  }
+`
+
+const ProposalsPage: React.FC<PageProps> = () => {
+  const {
+    grantsdao: { systemInfo, all, proposed, approved, completed, rejected },
+  } = useStaticQuery(PROPOSALS_PAGE_QUERY)
+  const columns = useMemo(
+    () => [
+      {
+        Header: "Title",
+        accessor: "description",
+        Cell: ({ row }) => (
+          <Link
+            to={"/proposals/" + row.original.number}
+            title={row.original.description}
+          >
+            <Text>
+              <strong>#{row.original.number}</strong> {row.original.description}
+            </Text>
+          </Link>
+        ),
+      },
+      {
+        Header: "Last Modified",
+        accessor: "modifiedAt",
+        Cell: ({ value }) => (
+          <Text title={fromUnixTime(value).toUTCString()}>
+            {formatDistanceToNow(fromUnixTime(value)).toUpperCase()} AGO
+          </Text>
+        ),
+      },
+      {
+        Header: "Created At",
+        accessor: "createdAt",
+        Cell: ({ value }) => (
+          <Text title={fromUnixTime(value).toUTCString()}>
+            {formatDistanceToNow(fromUnixTime(value)).toUpperCase()} AGO
+          </Text>
+        ),
+      },
+      {
+        Header: "",
+        accessor: "status",
+        Cell: ({ row }) => statusToBadge(row.original, systemInfo),
+      },
+      {
+        Header: "",
+        accessor: "voteCount",
+        Cell: ({ value }) => (
+          <Text style={{ whiteSpace: "nowrap" }}>
+            <strong>{value}</strong> VOTES
+          </Text>
+        ),
+      },
+    ],
+    []
+  )
+  const tablesInitialState = useMemo(
+    () => ({
+      sortBy: [{ id: "modifiedAt", desc: true }],
+    }),
+    []
+  )
+
+  // TODO: Remove after implement it, this effect just exists for logging purpose.
+  useEffect(() => {
+    console.log(
+      JSON.stringify(
+        { systemInfo, all, proposed, approved, completed, rejected },
+        null,
+        2
+      )
+    )
+  }, [systemInfo, all, approved, completed, rejected])
+
+  return (
+    <>
+      <SEO title="Proposals" />
```

Comment:
I think this title should change based on the current tab

Reply:
It is, the current tab is taken from the url if that's what you mean.